internal/tfengine/tfengine.go

```diff
@@ -225,6 +229,20 @@ func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateIn
 	return nil
 }
 
+func addGitIgnore(path string) error {
+	data := []byte(`# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+`)
+	if err := ioutil.WriteFile(fmt.Sprintf("%s/.gitignore", path), data, 0644); err != nil {
```

Comment:
This does not need execute permissions, 0444 might make more sense.

Reply:
0644 doesn't give any execution permissions https://chmodcommand.com/chmod-644/

Reply:
Oh right, my bad.